source/Cosmos.System2/Graphics/SVGAIICanvas.cs

```diff
@@ -138,6 +143,21 @@ public override void DrawPoint(Pen aPen, float aX, float aY)
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Crop the width of Shapes / Images / Etc
+        /// </summary>
+        public byte[] Crop(byte[] data, uint width, uint height, uint cropWidth, uint pixelSize)
```

Comment:
Why is this method in SVGAIICanvas? If we want to have such a method, it should be part of Bitmap/Image imo.

Reply:
good question, and idk lol. it can be moved